src/iOS/Avalonia.iOS/TextInputResponder.Properties.cs

```diff
@@ -81,7 +81,7 @@ public UIReturnKeyType ReturnKeyType
         [Export("smartInsertDeleteType")]
         public UITextSmartInsertDeleteType SmartInsertDeleteType { get; set; } = UITextSmartInsertDeleteType.Default;
 
-        [Export("passwordRules")] public UITextInputPasswordRules PasswordRules { get; set; } = null!;
+        [Export("passwordRules")] public UITextInputPasswordRules? PasswordRules { get; set; } = null!;
```

Comment:
`UITextInputPasswordRules.PasswordRules` is a nullable value for iOS and Catalyst. I'm not sure why it wasn't breaking for iOS normally, but adding Catalyst would break the build to complain about it not being nullable here. Adding the nullable flag fixes it.